style.css

```diff
@@ -88,21 +72,314 @@ h3 {
   line-height: 24px;
   color: #fefcfc;
 }
-/* stylelint-disable-next-line no-descending-specificity */
+
 .social-media a {
   padding: 1rem;
   color: white;
   font-size: 1.5rem;
   transition: 3s;
 }
 
+.nav a {
+  text-decoration: none;
+  color: #ffff;
+  padding: 0.7rem;
+  font-size: 1rem;
+}
+
 .social-media a:hover {
   text-shadow: #f87213;
   background: hwb(25 0% 0%);
   color: rgb(97, 91, 91);
   transition: 0.4s;
 }
 
-li {
-  display: inline;
+.nav a:hover {
+  background: hwb(25 0% 0%);
+  color: rgb(97, 91, 91);
+  transition: 0.4s;
+}
+
+.social {
+  display: flex;
+  justify-content: space-around;
+}
+
+.mainmyworks {
+  background-color: rgb(251, 249, 249);
+  padding-top: 40px;
+}
+
+.headtwo {
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Crete Round';
+  font-style: normal;
+  font-weight: 200;
+  font-size: 30px;
+  line-height: 52px;
+  margin-bottom: 25px;
+  display: flex;
+  justify-items: center;
+  color: #010101;
+  order: 0;
+  flex-grow: 0;
+  text-decoration: none;
+  padding-left: 20%;
+  padding-right: 10%;
+}
+
+.liner {
+  color: rgb(11, 0, 0);
+  border-bottom: none;
+  margin-bottom: 80px;
+}
+
+.workitems {
+  display: grid;
+  grid-template-rows: 1fr;
+  grid-template-columns: 1fr;
+  padding: 5%;
+}
+
+.photo img {
+  width: 100%;
+  position: relative;
+  padding-top: 0%;
+}
+
+.headfour {
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Crete Round';
+  font-style: normal;
+  font-weight: 200;
+  font-size: 25px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #01050c;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.page p {
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #00040a;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  padding-left: 0.01%;
+  padding-right: 0.01%;
+}
+
+.pagelinks {
+  display: flex;
+  width: 85%;
+  justify-content: space-around;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 6px;
+  padding: 8px;
+}
+
+.pagelinks li {
+  display: flex;
+  justify-content: space-around;
+  padding: 9px;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 200;
+  font-size: 12px;
+  line-height: 10px;
+  color: black;
+  border: #00040a solid 0.1px;
+  width: 20%;
+}
+
+.button {
+  background-color: #f65803;
+  padding: 12px;
+  border-radius: 12px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: rgb(255, 255, 254);
+  margin-top: 20px;
+  margin-bottom: 20px;
+  cursor: pointer;
+  width: fit-content;
+}
```

Comment:
- Please make sure that your project sticks as much as possible to the Figma template. :eyes: The button styling is different kindly make it similar.
- Also please change the font for the font provided in the Figma template. 
![image](https://user-images.githubusercontent.com/98363075/192100612-b5d040ca-44b5-4722-bd29-9effab49ccfe.png)